dangerzone/gui/main_window.py

```diff
@@ -59,13 +81,45 @@ def __init__(self, dangerzone: DangerzoneGui) -> None:
         header_version_label.setProperty("class", "version")
         header_version_label.setAlignment(QtCore.Qt.AlignBottom)
 
+        # Create the hamburger button, whose main purpose is to inform the user about
+        # updates.
+        self.hamburger_button = QtWidgets.QToolButton()
+        self.hamburger_button.setPopupMode(QtWidgets.QToolButton.InstantPopup)
+        self.hamburger_button.setIcon(
+            QtGui.QIcon(self.load_svg_image("hamburger_menu.svg"))
+        )
+        self.hamburger_button.setIconSize(QtCore.QSize(15, 15))
+        # FIXME: Maybe remove the box around the icon as well
+        self.hamburger_button.setStyleSheet(
+            "QToolButton::menu-indicator { image: none; }"
+        )
+        self.hamburger_button.setArrowType(QtCore.Qt.ArrowType.NoArrow)
+
+        # Create the menu for the hamburger button
+        hamburger_menu = QtWidgets.QMenu(self.hamburger_button)
+        self.hamburger_button.setMenu(hamburger_menu)
+
+        # Add the "Check for updates" action
+        self.toggle_updates_action = hamburger_menu.addAction("Check for updates")
+        self.toggle_updates_action.triggered.connect(self.toggle_updates_triggered)
+        self.toggle_updates_action.setCheckable(True)
+        self.toggle_updates_action.setChecked(
+            bool(self.dangerzone.settings.updater_get("check"))
+        )
+
+        # Add the "Exit" action
+        hamburger_menu.addSeparator()
+        exit_action = hamburger_menu.addAction("Exit")
+        exit_action.triggered.connect(self.close)
```

Comment:
I don't see a need for the exit action there. I think people are much more familiar with the window's :x: button.